client/coral-framework/utils/index.js

```diff
@@ -184,3 +184,11 @@ export function getShallowChanges(a, b) {
   return union(Object.keys(a), Object.keys(b))
     .filter((key) => a[key] !== b[key]);
 }
+
+export function getTotalReactionsCount(actionSummaries) {
+  return actionSummaries
+    .filter((s) => s.__typename !== 'FlagActionSummary')
```

Comment:
This is not a reliable method to determine `reactions`. There is not only `FlagActionSummary` but also `DontAgreeActionSummary` and on top of that not all plugin action summaries are sure to be reactions. What's your stance on this @wyattjoh?

Reply:
Seems fragile for me too as well. But I can't see other filtering option based on this arch. *Interested in what Wyatt has to say about this*

Reply:
I think we could use a GraphQL union type? But I'm a bit unsure..

Reply:
For now we can leave this as is. This might be a bigger task to classify all reaction types properly.